### ISSUE TITLE
Add `autocapitalize` attribute to input fields

### ISSUE THREAD

**Issue by mross-ua** [krazkidd/lineup#17]
For the team name and player name input text fields, we should instruct the browser to use an IME which auto-capitalizes words.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize

**Comment by VividhPandey003** [comment]
Hey, could you please assign me this issue under hacktoberfest

**Comment by mross-ua** [comment]
@VividhPandey003 You got it!

**Comment by VividhPandey003** [comment]
@krazkidd I am not able to locate the component where the changes need to be made. Could you please help me regarding that

**Comment by mross-ua** [comment]
@VividhPandey003 Sorry I wasn't clear enough. It is just these three components:

- [lineup page](https://github.com/krazkidd/lineup/blob/ba5971886b6ab36b766246e2d7b7182a9269600e/pages/lineup/index.vue#L33)
- [`LineupNewSpot`](https://github.com/krazkidd/lineup/blob/ba5971886b6ab36b766246e2d7b7182a9269600e/components/lineup/LineupNewSpot.vue#L41)
- [`LineupSpot`](https://github.com/krazkidd/lineup/blob/ba5971886b6ab36b766246e2d7b7182a9269600e/components/lineup/LineupSpot.vue#L45)

**Comment by VividhPandey003** [comment]
@krazkidd Sir, I have made the changes. By the changes I am referring to adding`autocapitalize="words"` to every input fields present in these three components. 

However, I am not able to setup the app on my local. To be honest I have not worked with Firebase until today. So I am not able to verify that the changes made are working or not.

If you could help me with setting up in my localhost, it would be greatly appreciated. However, the required changes are already done. You can check it here. If the required changes align with your need. I'll make a PR
[Branch Link](https://github.com/VividhPandey003/lineup/tree/autocapitalize)

**Comment by mross-ua** [comment]
@VividhPandey003 The changes are exactly what is needed! Go ahead and initiate a PR for the hackathon if you are okay licensing your contribution under the terms of the AGPL. Thank you! :smile: 

---

For the local setup, Google provides command line tools for running development versions of their cloud services. I have the setup steps documented [here](https://github.com/krazkidd/lineup/blob/main/README.md#setup) if you want to play around with Firestore (NoSQL). Basically, ou have to install the Firebase CLI package and then start the emulators before starting the app. You do not need a Firebase project to run the emulators.

**Comment by VividhPandey003** [comment]
> @VividhPandey003 The changes are exactly what is needed! Go ahead and initiate a PR for the hackathon if you are okay licensing your contribution under the terms of the AGPL. Thank you! 😄
> 
> For the local setup, Google provides command line tools for running development versions of their cloud services. I have the setup steps documented [here](https://github.com/krazkidd/lineup/blob/main/README.md#setup) if you want to play around with Firestore (NoSQL). Basically, ou have to install the Firebase CLI package and then start the emulators before starting the app. You do not need a Firebase project to run the emulators.

@krazkidd Thanks, I'll make a PR. btw what is AGPL. I didn't quite get you here- " if you are okay licensing your contribution under the terms of the AGPL."

Also could you please add "hacktoberfest-accepted" tag once you merge it

**Comment by mross-ua** [comment]
@VividhPandey003 The Affero GPL is one type of open source license. It requires that source code is provided when a program executable is shared or hosted on a website.

That means you can't give away compiled versions of my code without making the source available and *I* can't use your code without making the source available, as it is a strong copyleft license. Most open source projects use a license which allows them to use contributions from others without giving anything back.

**Comment by VividhPandey003** [comment]
> @VividhPandey003 The Affero GPL is one type of open source license. It requires that source code is provided when a program executable is shared or hosted on a website.
> 
> That means you can't give away compiled versions of my code without making the source available and _I_ can't use your code without making the source available, as it is a strong copyleft license. Most open source projects use a license which allows them to use contributions from others without giving anything back.

Oh, I just knew about MIT License. Thanks @krazkidd, you're really welcoming. By the way I am also learning Web Development these days. Would love to contribute to more repos along with you. Although the issue was really easy to be solved, I got to learn something new. Thanks🚀

**Comment by mross-ua** [comment]
Thank you. I started this project to learn Vue.js/Nuxt but I would recommend you go with React/Next if you are looking to be employed as a web dev in the near future. I'm trying to think of an app I can build with React...

**Comment by VividhPandey003** [comment]
> Thank you. I started this project to learn Vue.js/Nuxt but I would recommend you go with React/Next if you are looking to be employed as a web dev in the near future. I'm trying to think of an app I can build with React...

I am actually learning React these days. I am an undergrad so kinda learning it when i'm kinda free/ don't have any assignments. Would ping you for sure if I get any cool react projects.
